lib/blacklight/configuration.rb

```diff
@@ -212,7 +212,7 @@ def repository_class
     end
 
     def search_builder_class
-      super || Blacklight::Solr::SearchBuilder
+      super || ::SearchBuilder
```

Comment:
Can we do this and still claim backwards compatibility, or do we need to check if it is defined first?